src/complex/Parameters/DataStoreFormatParameter.cpp

```diff
@@ -0,0 +1,81 @@
+#include "DataStoreFormatParameter.hpp"
+
+#include "complex/Common/Any.hpp"
+#include "complex/Core/Application.hpp"
+
+#include <fmt/core.h>
+#include <nlohmann/json.hpp>
+
+namespace
+{
+constexpr complex::int64 k_BadFormatCode = -780;
+}
+
+namespace complex
+{
+DataStoreFormatParameter::DataStoreFormatParameter(const std::string& name, const std::string& humanName, const std::string& helpText, const ValueType& defaultValue)
+: ValueParameter(name, humanName, helpText)
+, m_DefaultValue(defaultValue)
+{
+}
+
+Uuid DataStoreFormatParameter::uuid() const
+{
+  return ParameterTraits<DataStoreFormatParameter>::uuid;
+}
+
+IParameter::AcceptedTypes DataStoreFormatParameter::acceptedTypes() const
+{
+  return {typeid(ValueType)};
+}
+
+nlohmann::json DataStoreFormatParameter::toJson(const std::any& value) const
+{
+  const auto& stringValue = GetAnyRef<ValueType>(value);
+  nlohmann::json json = stringValue;
+  return json;
```

Comment:
```suggestion
  return GetAnyRef<ValueType>(value);
```
just return the value directly